xcclookup/tools/tools.go

```diff
@@ -0,0 +1,17 @@
+//go:build tools
+// +build tools
```

Comment:
`// +build tools`
legacy  syntax only one of the above lines is needed to point to use when `tools` build tag is used